extensions/amp-animation/0.1/amp-animation.js

```diff
@@ -42,17 +60,125 @@ export class AmpAnimation extends AMP.BaseElement {
     user().assert(isExperimentOn(this.win, TAG),
         `Experiment "${TAG}" is disabled.`);
 
+    // Trigger.
+    const trigger = this.element.getAttribute('trigger');
+    if (trigger) {
+      this.triggerOnVisibility_ = user().assert(
+          trigger == 'visibility',
```

Comment:
I'm a bit confused by what `visibility` means here and what the goal of the PR is. Is this `document-visibility`?